meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -44,14 +48,25 @@ OSTRO_VM_IMAGE_TYPES = "dsk.xz dsk.zip dsk.vdi.zip"
 # which must contain only alphanumeric symbols,'-' and '_'.
 # Any other symbols would be skipped in parser.
 #
-# Following targets would be used to perform default build task:
-OSTROPROJECT_CI_BUILD_TARGETS="${DISTRO}-image ${DISTRO}-image-dev"
+# Following targets would be used to perform default build task.
+OSTROPROJECT_CI_BUILD_TARGETS="ostro-image ostro-image-qa ostro-image-reference"
 # Following targets would be executed with do_populate_sdk task
 OSTROPROJECT_CI_SDK_TARGETS=""
-# Following targets would be executed with do_populate_sdk_ext task
-OSTROPROJECT_CI_ESDK_TARGETS="${DISTRO}-image"
-# Following targets would be executed with do_test_iot_export task
-OSTROPROJECT_CI_TEST_EXPORT_TARGETS="${DISTRO}-image"
+# Following targets would be executed with do_populate_sdk_ext task.
+# Uses the fully-populated image to ensure that the eSDK is as complete
+# as possible.
+OSTROPROJECT_CI_ESDK_TARGETS="ostro-image-all"
+# Following targets would be executed with do_test_iot_export task.
+# Must be "ostro-image", the task seems to expect that name.
+OSTROPROJECT_CI_TEST_EXPORT_TARGETS="ostro-image"
+
+# ostro-image as build target above shall only produce bundles,
+# but no images.
+OSTRO_VM_IMAGE_TYPES_pn-ostro-image = ""
+IMAGE_FSTYPES_pn-ostro-image = ""
```

Comment:
Point me to exact mail in this thread where it was proposed different content for automated and for user builds.
I don't have time to re-read 68 mails to find one to reply to.
Code wise, for this chunk I don't agree.